lib/recurly/client.rb

```diff
@@ -53,45 +53,36 @@ def initialize(api_key:, site_id: nil, subdomain: nil, **options)
       yield(self) if block_given?
     end
 
-    def next_page(url)
```

Comment:
For anyone watching, we decided to refactor this functionality by making it private / shared.

Reply:
The Pager now calls the private API of Client